kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -112,6 +115,8 @@ def _run(self, target_api_config_path, auto_transform=False,
                 transform_fp = os.path.join(
                     self.ingest_config_dir, os.path.relpath(transform_fp))
 
+        id_cache_filepath = os.path.join(self.ingest_config_dir,
```

Comment:
I'm assuming the ID cache file will be stored in the ingest config dir. If this is a bad assumption then, we'll need to have `id_cache_filepath` as a parameter in `dataset_ingest_config.yml` and use the FileRetriever to fetch the file from wherever it is.

Figured reading it from ingest config dir for the first go around would be fine

Reply:
agree